src/ralph/virtual/models.py

```diff
@@ -252,6 +252,7 @@ class VirtualServer(AdminAbsoluteUrlMixin, NetworkableBaseObject, BaseObject):
         verbose_name=_('SN'),
         unique=True,
     )
+    # TODO: remove this field
```

Comment:
I meant.. "add some hint to others, that they could delete it, if condition will meet" :)
of if you handle that then it can be ignored